pycode/memilio-surrogatemodel/memilio/surrogatemodel/GNN/data_generation_nodamp.py

```diff
@@ -0,0 +1,328 @@
+                InfectionState.InfectedSymptomsConfirmed)] = 0
+            model.populations[AgeGroup(i), Index_InfectionState(
+                InfectionState.InfectedSevere)] = random.uniform(
+                0.00003, 0.0006) * populations[region][i]
+            model.populations[AgeGroup(i), Index_InfectionState(
+                InfectionState.InfectedCritical)] = random.uniform(
+                0.00001, 0.0002) * populations[region][i]
+            model.populations[AgeGroup(i), Index_InfectionState(
+                InfectionState.Recovered)] = random.uniform(
+                0.002, 0.08) * populations[region][i]
+            model.populations[AgeGroup(i),
+                              Index_InfectionState(InfectionState.Dead)] =  random.uniform(
+                0, 0.0003) * populations[region][i]
+            model.populations.set_difference_from_group_total_AgeGroup(
+                (AgeGroup(i), Index_InfectionState(InfectionState.Susceptible)),  
+                    populations[region][i])
+
+            # Compartment transition propabilities
+            model.parameters.RelativeTransmissionNoSymptoms[AgeGroup(i)] = 0.5
+            model.parameters.TransmissionProbabilityOnContact[AgeGroup(
+                i)] = 0.1
+            model.parameters.RecoveredPerInfectedNoSymptoms[AgeGroup(i)] = 0.09
+            model.parameters.RiskOfInfectionFromSymptomatic[AgeGroup(i)] = 0.25
+            model.parameters.SeverePerInfectedSymptoms[AgeGroup(i)] = 0.2
+            model.parameters.CriticalPerSevere[AgeGroup(i)] = 0.25
+            model.parameters.DeathsPerCritical[AgeGroup(i)] = 0.3
+            # twice the value of RiskOfInfectionFromSymptomatic
+            model.parameters.MaxRiskOfInfectionFromSymptomatic[AgeGroup(
+                i)] = 0.5
+
+        # StartDay is the n-th day of the year
+        model.parameters.StartDay = (
+            date(start_year, start_month, start_day) - date(start_year, 1, 1)).days
+
+        # Load baseline and minimum contact matrix and assign them to the model
+        baseline = getBaselineMatrix()
+        #minimum = getMinimumMatrix()
+
+        model.parameters.ContactPatterns.cont_freq_mat[0].baseline = baseline
+        model.parameters.ContactPatterns.cont_freq_mat[0].minimum = np.ones(
+        (num_groups, num_groups)) * 0
+
+        # Apply mathematical constraints to parameters
+        model.apply_constraints()
+        models.append(model)
+
+    graph = ModelGraph()
+    for i in range(num_regions):
+        graph.add_node(int(countykey_list[i]), models[i])
+
+    # get mobility data directory
+    arg_dict = gd.cli("commuter_official")
+
+    directory = arg_dict['out_folder'].split('/pydata')[0]
+    directory = os.path.join(directory, 'mobility/')  
+
+    # Merge Eisenach and Wartbugkreis in Input Data 
+    tmd.updateMobility2022(directory, mobility_file='twitter_scaled_1252')
+    tmd.updateMobility2022(directory, mobility_file='commuter_migration_scaled')
```

Comment:
good way to solve this. But this just need to be called once. So for the second, third,... call of this function this is no necessary. Move this to a own function and call this if needed there.